makefile

```diff
@@ -32,20 +32,27 @@ default:
 	@echo "usage: make target"
 	@echo "available targets: compile, test, clean"
 
-compile: javatrix/Matrix.class junit/MatrixTest.class
+compile: javatrix/Matrix.class junit/MatrixTest.class junit/ArithmaticTests.java junit/InPlaceArithmaticTests.java
 	@echo "compiled"
-	
+
 junit/MatrixTest.class: $(JUNIT_LOCAL)
+junit/ArithmaticTest.class: $(JUNIT_LOCAL)
+junit/InPlaceArithmaticTest.class: $(JUNIT_LOCAL)
 
 style:
 	checkstyle -c $(STYLE_XML) junit/MatrixTest.java javatrix/Matrix.java
```

Comment:
If we're going to test one file, we should test all of them. This should probably say `junit/*.java`. That being said, this PR has enough change in it already.